src/lib.rs

```diff
@@ -64,6 +64,8 @@ extern crate alloc;
 
 use alloc::vec::Vec;
 use core::mem::ManuallyDrop;
+use core::hash::{Hash, Hasher}; 
+use core::fmt;
```

Comment:
Can you alphabetize these imports?

Reply:
Sure! I must've added those in the order I worked on the impls but I'll alphabetize them instead.